docs/src/rules/prefer-destructuring.md

```diff
@@ -16,20 +16,28 @@ With JavaScript ES6, a new syntax was added for creating variables from an array
 
 This rule takes two sets of configuration objects. The first object parameter determines what types of destructuring the rule applies to.
```

Comment:
I think the use of the word "sets" is incorrect throughout. What we are really saying is that this rule takes two arguments, both of which are objects. 

Then we can say, "The first object contains two properties, `array` and `object`..."

And, "The second object..."

What do you think?

Reply:
i think it explains the option well,
updated the description!